common/network-defaults/src/lib.rs

```diff
@@ -107,17 +107,19 @@ pub fn default_api_endpoints() -> Vec<Url> {
 }
 
 pub const ETH_CONTRACT_ADDRESS: [u8; 20] =
-    hex_literal::hex!("9fEE3e28c17dbB87310A51F13C4fbf4331A6f102");
+    hex_literal::hex!("8e0DcFF7F3085235C32E845f3667aEB3f1e83133");
 // Name of the event triggered by the eth contract. If the event name is changed,
 // this would also need to be changed; It is currently tested against the json abi
-pub const ETH_EVENT_NAME: &str = "Burned";
-pub const ETH_BURN_FUNCTION_NAME: &str = "burnTokenForAccessCode";
+pub const ETH_EVENT_NAME: &str = "BBCredentialPurchased";
```

Comment:
per our discussion, should the event be "purchased" or rather "converted" or something along those lines?